app/components/pdf_component.html.erb

```diff
@@ -11,6 +11,7 @@
     <div role="banner" aria-label="Access message">
       <%= render LoginComponent.new %>
       <%= render CompanionWindows::ContentNotAvailableComponent.new unless viewer.available? %>
+      <%= render LocationRestrictionComponent.new(viewer.restricted_location) if viewer.location_restricted? %>
```

Comment:
Can't we display this all of the time, without the check, and wait for the front end to say "yes location restricted".  Otherwise we are checking once on the front end and once on the back end.  That doesn't seem very efficient.